src/linter/linter.go

```diff
@@ -34,10 +33,10 @@ func (l *Linter) NewIndexingWorker(id int) *Worker {
 
 // AnalyzeFiles runs linter on the files that are provided by the readFileNamesFunc function.
 func (l *Linter) AnalyzeFiles(readFileNamesFunc workspace.ReadCallback) []*Report {
-	return l.analyzeFiles(readFileNamesFunc, l.config.AllowDisable)
+	return l.analyzeFiles(readFileNamesFunc)
 }
 
-func (l *Linter) analyzeFiles(readFileNamesFunc workspace.ReadCallback, allowDisable *regexp.Regexp) []*Report {
+func (l *Linter) analyzeFiles(readFileNamesFunc workspace.ReadCallback) []*Report {
```

Comment:
See another comment. I'm not sure this method should be changed.
At least it's not right to change it as a part of PHP extensions related PR.